### ISSUE TITLE
Do not display soft-deleted records unless specfied

### ISSUE THREAD

**Issue by shubham-padia** [fossasia/open-event-orga-server#3696]
From https://github.com/fossasia/open-event-orga-server/issues/3686#issuecomment-304776817
```
And for the GET endpoints... By default soft deleted records will not be
retrieved. But when passing a query parameter like with_trashed=true, soft
deleted should also be included.

And for direct ID retrievals
If for example, in case of an event.. Let's say event 1 is soft deleted.
/events/1 - 404 error
/events/1?with_trashed=true - get that event record
```

**Comment by shubham-padia** [comment]
@niranjan94 The api framework provides filtering, http://flask-rest-jsonapi.readthedocs.io/en/latest/filtering.html
with which the client could make a request of their choice. 
As I'm not familiar with Ember, is there any provision in ember that would make handling this client side with the help of filtering easier ?

**Comment by niranjan94** [comment]
@shubham-padia soft-deletes aren't supposed to be handled from the client side. Soft-delete means the data is gone. It's shouldn't be there unless required.

**Comment by niranjan94** [comment]
This should be handled in the backend.

**Comment by shubham-padia** [comment]
Okay.

**Comment by shubham-padia** [comment]
closed via #3702
